### PR TITLE
akamai prefix removed from classes

### PR DESCRIPTION
**Pull request by neetumorwani**

### CODE REVIEW

**Comment by nerdstein** [comment]
Looks good, merging

**Comment by nerdstein** [comment]
@cam8001 do you want to do a code review first?

**Comment by cam8001** [comment]
There are remaining classes that still have the prefix - AkamaiClient, AkamaiAuthentication, for example, that extend the upstream Akamai client. Do we want to rename those too?

**Comment by nerdstein** [comment]
They are not under conventional core namespaced directories, so I'm not sure it makes sense to remove the prefix from those. Class names for generic classes in src may get confusing. But, I'm not sure exactly what the convention is for those.

**Comment by cam8001** [comment]
I'm cool with that, let's merge.

File: .travis.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@ env:
 
     # Configuration variables.
     - DRUPAL_TI_MODULE_NAME="akamai"
-    # - DRUPAL_TI_SIMPLETEST_GROUP="akamai"
+    - DRUPAL_TI_SIMPLETEST_GROUP="akamai"
```
Comment (naveenvalecha) [inline .travis.yml:25]: This should be the **Akamai**

Comment (cam8001) [inline .travis.yml:25]: Yep. I am fiddling with travis on a branch over here if you want to have a play:  https://github.com/cam8001/akamai/tree/travis-updates
https://travis-ci.org/cam8001/akamai

Comment (naveenvalecha) [inline .travis.yml:25]: Yup I'll check my own github fork on travis ci

**Pull request merged**